core/tests/protractor_desktop/skillEditor.js

```diff
@@ -80,19 +80,21 @@ describe('Skill Editor functionality', function() {
   });
 
   it('should create and delete worked examples', function() {
-    skillEditorPage.addWorkedExample('Example 1');
-    skillEditorPage.addWorkedExample('Example 2');
+    skillEditorPage.addWorkedExample(
+      'Example Question 1', 'Example Explanation 1');
+    skillEditorPage.addWorkedExample(
+      'Example Question 2', 'Example Explanation 2');
     skillEditorPage.saveOrPublishSkill('Added worked examples');
 
     skillEditorPage.get(skillId);
     skillEditorPage.expectWorkedExampleSummariesToMatch([
-      'Example 1', 'Example 2']);
+      'Example Question 1', 'Example Question 2']);
```

Comment:
Can we also check the explanations? We could split `expectWorkedExampleSummariesToMatch` into a function for checking the questions and a function for checking the explanations.

Reply:
Done